src/app/plan/duck/reducers.ts

```diff
@@ -34,7 +35,10 @@ export const INITIAL_STATE = {
     state: CurrentPlanState.Pending,
     errorMessage: ''
   } as ICurrentPlanStatus,
-  lockedPlanList: []
+  lockedPlanList: [],
+  isFetchingHookList: false,
+  migHookList: [],
+  hookAddEditStatus: defaultAddEditStatus()
 };
```

Comment:
In the future we should probably make type interfaces for the initial state in each of these reducers

Reply:
+1 this is used across the app. TS support would be nice.